sycl/include/sycl/ext/oneapi/experimental/graph.hpp

```diff
@@ -378,17 +372,30 @@ class __SYCL_EXPORT modifiable_command_graph {
   /// added as dependencies.
   void addGraphLeafDependencies(node Node);
 
+  void print_graph(sycl::detail::string_view path, bool verbose = false) const;
+
   template <class Obj>
   friend const decltype(Obj::impl) &
   sycl::detail::getSyclObjImpl(const Obj &SyclObject);
   template <class T>
   friend T sycl::detail::createSyclObjFromImpl(decltype(T::impl) ImplObj);
-#ifdef __INTEL_PREVIEW_BREAKING_CHANGES
-  void print_graph(sycl::detail::string_view path, bool verbose = false) const;
-#endif
   std::shared_ptr<detail::graph_impl> impl;
 };
 
+#ifdef __SYCL_GRAPH_IMPL_CPP
+__SYCL_EXPORT
+#if WIN32
```

Comment:
I am not sure I understand this part, could you please clarify why have  inline in cpp file only on windows?